code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -198,9 +198,7 @@
 		if(istype(W, /obj/item/ego_weapon))
 			var/obj/item/ego_weapon/EW = W
 			if(!EW.CanUseEgo(src))
-				to_chat(user, span_notice("You try attaching \the [W] to \the [src]... but it falls off!"))
-				user.dropItemToGround(EW)
-				EW.forceMove(get_turf(src))
+				to_chat(user, span_notice("You try attaching \the [W] to \the [src]... but it refuses to stay on!"))
```

Comment:
imo, this is a good PR but the old message was better

Reply:
This message does make more sense though